doc/developer-notes.md

```diff
@@ -831,11 +831,6 @@ int GetInt(Tabs tab)
 Strings and formatting
 ------------------------
 
-- Be careful of `LogPrint` versus `LogPrintf`. `LogPrint` takes a `category` argument, `LogPrintf` does not.
-
-  - *Rationale*: Confusion of these can result in runtime exceptions due to
```

Comment:
I don't even understand how this can lead to runtime errors. Trying to match an enum to a format string should never succeed at compile time.

cc @laanwj who added this in commit b8c06ef409792dd9a6d14d46b50787fa7a6fb33d.

Reply:
Category was a string, not an enum back in 2015.
```
src/addrman.cpp:    LogPrint("addrman", "Moving %s to tried\n", addr.ToString());
```
So it was possible to get it wrong. I agree this is not longer the case as it is type-safe now.

Reply:
Thanks for the context!